frontend/src/lib/k8s/cluster.ts

```diff
@@ -330,554 +294,25 @@ export interface AuthRequestResourceAttrs {
   group?: string;
   verb?: string;
 }
-type JsonPath<T> = T extends object
-  ? {
-      [K in keyof T]: K extends string ? `${K}` | `${K}.${JsonPath<T[K]>}` : never;
-    }[keyof T]
-  : never;
-
-// @todo: uses of makeKubeObject somehow end up in an 'any' type.
 
 /**
+ * @deprecated For backwards compatibility, please extend KubeObject
```

Comment:
I think you need a period where you added a comma. Otherwise it reads as a the solution for the backwards compatibility.